test/WebSites/MvcTagHelpersWebSite/Controllers/MvcTagHelper_HomeController.cs

```diff
@@ -12,22 +12,24 @@ namespace MvcTagHelpersWebSite.Controllers
 {
     public class MvcTagHelper_HomeController : Controller
     {
-        public IActionResult Order()
-        {
-            var products = new List<Product>();
-            products = new List<Product>();
+        private readonly List<Product> _products = new List<Product>();
 
+        public MvcTagHelper_HomeController()
+        {
             for (int i = 7; i < 13; ++i)
             {
-                products.Add(new Product()
+                _products.Add(new Product()
                 {
                     ProductName = "Product_" + i,
                     Number = i,
                     PartNumbers = Enumerable.Range(1, 3).Select(n => string.Format("{0}-{1}", i, n))
```

Comment:
why does `Product` have `HomePage` and `Description` properties if none of these entries set them?